src/services/youtube.js

```diff
@@ -59,8 +59,7 @@ function genAsyncGetFeedsFn(url) {
     ytdl = youtubedl.exec;
   }
   return () =>
-    ytdl(null, {
-      "--": [url],
```

Comment:
See, this is quite necessary. Context: https://github.com/miraclx/freyr-js/issues/83

```suggestion
    ytdl(null, {
      "--": [url],
```